<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firewall_window</class>
 <widget class="QWidget" name="firewall_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firewall</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/prefix2/resources/linux.svg</normaloff>:/new/prefix2/resources/linux.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="services_page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="to_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Allowed services </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="services_layout">
         <item row="0" column="1">
          <layout class="QGridLayout" name="interface_layout">
           <item row="0" column="1">
            <widget class="QComboBox" name="allowed_interface_services"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Interface (not necessary?)</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="allowed_services_ok">
             <property name="text">
              <string>Ok</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="add_output_rule">
             <property name="text">
              <string>Add Otuput Rule</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="3" column="0">
            <widget class="QCheckBox" name="https_check">
             <property name="text">
              <string>HTTPS</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="allowed_services_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Allows to mark these services to allowed trafic for interface, only adding allowed at this point</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="sql_check">
             <property name="text">
              <string>MySQL</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="http_check">
             <property name="text">
              <string>HTTP</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="ftp_check">
             <property name="text">
              <string>FTP</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QGridLayout" name="rules_layout">
           <item row="1" column="1">
            <widget class="QPushButton" name="list_rules_ok">
             <property name="text">
              <string>List</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="list_rules_chain">
             <item>
              <property name="text">
               <string>INPUT</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>OUTPUT</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>FORWARD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NAT</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="list_rules_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>List rules for chain</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QGridLayout" name="custom_port_layout">
           <item row="0" column="0">
            <widget class="QLabel" name="custom_port_label">
             <property name="text">
              <string>Custom port:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="custom_port_interface">
             <item>
              <property name="text">
               <string>all-interfaces</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="custom_port_value">
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDialogButtonBox" name="button_box">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="Line" name="services_line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="routing_dnat_page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="local_interface_label">
           <property name="text">
            <string>Local interface:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="local_interface"/>
         </item>
         <item>
          <widget class="QCheckBox" name="public_ip_checkbox">
           <property name="text">
            <string>Public IP?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="internet_interface_label">
           <property name="text">
            <string>Internet:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="internet_interface"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="routing_apply">
           <property name="text">
            <string>Apply</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="dnat_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>DNAT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="routing_dnat_line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="dnat_ip_label">
           <property name="text">
            <string>Source IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="source_ip"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Destination IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="destination_ip"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="dnat_port_label">
           <property name="text">
            <string>Source port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="source_port"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string> Destination port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="destination_port"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="dnat_interface_label">
           <property name="text">
            <string>Source interface</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="source_interface"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Destination interface</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="destination_interface"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="dnat_apply">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="routing_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>ROUTING</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
